specs/pvs/corla.pvs

```diff
@@ -347,7 +377,10 @@ END election
 % @relation box inherit storage_container
```

Comment:
Yes, @nealmcb captured my intention.  The general idea of a ballot is that it convey's the meaning of a voter's intent, but by no means is it actually a record of voter intent, since so many kinds of ballots are not voter-verifiable.